lib/utils.c

```diff
@@ -849,3 +849,27 @@ sr_demangle_symbol(const char *sym)
 
     return demangled;
 }
+
+char *
+anonymize_filename(const char *file_name)
+{
+    if (strncmp(file_name, "/home/", 6) == 0) {
+      // Find how rest after /home/.../
+      for (int i = 0; i < 3; i++){
```

Comment:
Codding style.

Reply:
Why are you iterating here and why 3 times?

Reply:
/home/user/rest
^.......^......^  I need to go after third /.

Reply:
In this `if` branch you know `file_name` starts with `"/home/"`. You can use just, for instance:
```
file_name = strchr(file_name + strlen("/home/"), '/');
```